occupancy_sensing.cpp

```diff
@@ -212,20 +212,6 @@ void DeRestPluginPrivate::handleOccupancySensingClusterIndication(const deCONZ::
                 configUpdated = true;
             }
 
-            if (sensor->mustRead(WRITE_SENSITIVITY))
-            {
-                ResourceItem *item = sensor->item(RConfigPending);
-                if (item)
-                {
-                    quint16 mask = item->toNumber();
-                    mask &= ~R_PENDING_SENSITIVITY;
-                    item->setValue(mask);
-                    Event e(RSensors, RConfigPending, sensor->id(), item);
-                    enqueueEvent(e);
-                }
-                sensor->clearRead(WRITE_SENSITIVITY);
-            }
-
```

Comment:
Here, I'd also say the while case can be removed?

Reply:
Indeed, both `HUE_SENSITIVITY` and `HUE_SENSITIVITY_MAX` are Hue-specific.